napari/_qt/event_loop.py

```diff
@@ -22,6 +25,16 @@ def gui_qt(*, startup_logo=False):
     IPython with the Qt GUI event loop enabled by default by using
     ``ipython --gui=qt``.
     """
+    if platform.system() == "Windows" and not getattr(sys, 'frozen', False):
```

Comment:
I'm not sure we plan on using pyinstaller anymore... is `frozen` used for anything else?

Reply:
There is #496 work to create bundle app. I try it also in #1082 but fail on some openGL problem. I'm not on time with napari talks, so maybe you change plan. 

In my case I need to create bundle, to avoid setup python by some of users.

Reply:
yep I'm aware of #496 (look at the commit history there).  and yes, we're considering alternatives that will make it easier to extend the environment after bundling.  lots of discussions going on here https://github.com/napari/napari/issues/1074

if you are creating a bundle, I assume it is of your own application that wraps napari right?  In which case wouldn't you be using your own icon?

Reply:
When I create bundle there is no problem with icon, because program is not interpreted as python one. Problem is in this line:
https://github.com/napari/napari/blob/9ed0fda58d2dca7f14e67107a0fd5b3791116555/napari/_qt/event_loop.py#L34-L36

This may cause that two apps which using napari viewer will be treated as one because they have same SetCurrentProcessExplicitAppUserModelID